mask/eolearn/mask/cloud_mask.py

```diff
@@ -292,3 +297,592 @@ def get_s2_pixel_cloud_detector(threshold=0.4, average_over=4, dilation_size=2,
+                                      Resolution is given as a pair of x and y resolutions. If a single value is
+                                      given, it is used for both dimensions.
+                                      Default is `None` (source resolution).
+        :type processing_resolution: (str, str) or (int, int) or str or int
+        :param max_proc_frames: Maximum number of frames (including the target, for multi-temporal classification)
+                                considered in a single batch iteration (To keep memory usage at agreeable levels,
+                                the task operates on smaller batches of time frames). Default value:  `11`.
+        :type max_proc_frames: int
+        :param mono_features: Tuple of keys to be used for storing cloud probabilities and masks of the mono classifier.
+                              The probabilities are added to the `eopatch.data` attribute dictionary, while masks are
+                              added to `eopatch.mask`. By default, none of them are added.
+        :type mono_features: (str | None, str | None)
+        :param multi_features: Tuple of keys used for storing cloud probabilities and masks of the multi classifier.
+                               The probabilities are added to the `eopatch.data` attribute dictionary, while masks are
+                               added to `eopatch.mask`. By default, none of them are added.
+        :type multi_features: (str | None, str | None)
+        :param mask_feature: Name of the output intersection feature. The masks are added to the `eopatch.mask`
+                             attribute dictionary. Default value: `'CLM_INTERSSIM'`. If None, the intersection
+                             feature is not computed.
+        :type mask_feature: str | None
+        :param mono_threshold: Cloud probability threshold for the mono classifier. Default value: `0.4`.
+        :type mono_threshold: float
+        :param multi_threshold: Cloud probability threshold for the multi classifier. Default value: `0.5`.
+        :type multi_threshold: float
+        :param average_over: Size of the pixel neighbourhood used in the averaging post-processing step.
+                             A value of `0` or `None` skips this post-processing step. Default value: `1`.
+        :type average_over: int or None
+        :param dilation_size: Size of the dilation post-processing step. A value of `0` or `None` skips
+                              this post-processing step. Default value: `1`.
+        :type dilation_size: int or None
+        """
+
+        # Load classifiers
+        if mono_classifier is None or multi_classifier is None:
+            classifier_dir = os.path.dirname(__file__)
+
+            if mono_classifier is None:
+                mono_classifier = joblib.load(os.path.join(classifier_dir, 'models', MONO_CLASSIFIER_NAME))
```

Comment:
This line fails for me if I do the following in a jupyter notebook:

```python
from eolearn.core import EOPatch, LoadFromDisk
from eolearn.mask import AddCloudMaskTask, get_s2_pixel_cloud_detector, AddMultiCloudMaskTask

eop = EOPatch.load('/mnt/s3/turlcms/tr-project/2018/L1C/data/yearly/eopatch-id-468-col-14-row-58', lazy_loading=True)
add_multi = AddMultiCloudMaskTask(processing_resolution='120m',
                                  mask_feature='CLM_INTERSSIM',
                                  average_over=16,
                                  dilation_size=8)
```

Probably the model pickle version is too old? Because for the lower one it works

Reply:
What do you mean by the lower one?

Reply:
Ok now the tests fail on Travis as well. It seems the libraries were updated and the model can't be read anymore.

Reply:
I checked that the models were pickled with two different versions of pickle and the one that fails seems to be too old. Did you manage to pinpoint this, otherwise I can give you more information.

Reply:
OK, here is some more info. I Try this:

```python
import os
import joblib

DIR = '/home/users/mlubej/projects/eo-learn/mask/eolearn/mask/models/'
MONO_CLASSIFIER_NAME = 'pixel_s2_cloud_detector_lightGBM_v0.1.joblib.dat'
MULTI_CLASSIFIER_NAME = 'ssim_s2_cloud_detector_lightGBM_v0.2.joblib.dat'

models = [MONO_CLASSIFIER_NAME, MULTI_CLASSIFIER_NAME]

for model in models:
    try:
        joblib.load(os.path.join(DIR, model))
        print(f'Loaded {model} successfully!')
    except:
        print(f'Failed to load {model}')
```

and the output is:

```
Failed to load pixel_s2_cloud_detector_lightGBM_v0.1.joblib.dat
Loaded ssim_s2_cloud_detector_lightGBM_v0.2.joblib.dat successfully!
```

While the error for loading the `pixel_s2_cloud_detector_lightGBM_v0.1.joblib.dat` is:

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-17-090eb69ef74f> in <module>
----> 1 joblib.load(os.path.join(DIR, MONO_CLASSIFIER_NAME))

~/base/lib/python3.6/site-packages/joblib/numpy_pickle.py in load(filename, mmap_mode)
    596                     return load_compatibility(fobj)
    597 
--> 598                 obj = _unpickle(fobj, filename, mmap_mode)
    599 
    600     return obj

~/base/lib/python3.6/site-packages/joblib/numpy_pickle.py in _unpickle(fobj, filename, mmap_mode)
    524     obj = None
    525     try:
--> 526         obj = unpickler.load()
    527         if unpickler.compat_mode:
    528             warnings.warn("The file '%s' has been generated with a "

/usr/lib/python3.6/pickle.py in load(self)
   1048                     raise EOFError
   1049                 assert isinstance(key, bytes_types)
-> 1050                 dispatch[key[0]](self)
   1051         except _Stop as stopinst:
   1052             return stopinst.value

KeyError: 0
```

Reply:
Thanks! We updated the model pickle version.

Reply:
Can you test if it works for you now?

Reply:
The code now runs for me and the model loads, so that's fixed.

Can you look into why the Travis build is failing?

Also, do you know why running `pytest` didn't give this error? That's weird.

Reply:
Travis build fails on this `The command "if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then pytest --ignore io/eolearn/tests; fi" exited with 1.`

Looks like a problem in `io` module? 
Not sure why, but Travis builds started failing this week. Code changes were minor and shouldn't break the tests.

Reply:
@AleksMat any ideas?